Readme.md

```diff
@@ -1,25 +1,111 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128534875/19.2.4%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1760)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to implement select/unselect for all rows in a group row
+# Grid View for ASP.NET Web Forms - How to select/deselect all rows in a group when the control is grouped by a column
```

Comment:
```suggestion
# Grid View for ASP.NET Web Forms - How to select/deselect all rows in a group when a control is grouped by columns
```

Reply:
The technique described in this example works only when the Grid is grouped by one column. Also, when I say "the control", I mean "the Grid View control", that's why I use the definite article.
Maybe we can say the following?

```suggestion
# Grid View for ASP.NET Web Forms - How to select/deselect all rows in a group when the control is grouped by one column
```

Reply:
Hello Armina,

Please excuse me for interfering, but I should have noticed this in the first place. One groups data, not the control itself. Regarding one column -- yes, this is the important part of the described scenario. I hope the following change makes sense:

```suggestion
# Grid View for ASP.NET Web Forms - How to select/deselect all rows in a group when data is grouped by one column
```

Reply:
yes

Reply:
Thank you!